engine.py

```diff
@@ -27,7 +27,17 @@ class HoudiniEngine(tank.platform.Engine):
     """
     Houdini Engine implementation
     """
-    
+
+    @property
+    def host_info(self):
+        """
+        :returns: A {"name": application name, "version": application version}
+                  dictionary with informations about the application hosting this
+                  engine.
+        """
+        hou_ver_str = ".".join([str(v) for v in hou.applicationVersion()])
+        return {"name": "Houdini", "version": hou_ver_str}
```

Comment:
check `hou.applicationName()` for the `name` value. There are different houdini "products" that Toolkit can run in (Houdini FX, Houdini Apprentice, Houdini Core, etc.)

http://www.sidefx.com/docs/houdini/hom/hou/applicationName

There's also `hou.applicationVersionString()` for the version string. 

http://www.sidefx.com/docs/houdini/hom/hou/applicationVersionString

Looks like these methods existed all the way back in Houdini 14, so should be ok to use assuming they return what we want/expect.

Reply:
Just when I though I could do something in 5 minute! I do appreciate you being thorough.

Tested (On Mac) both apprentice and FX flavors. Athough the splash and about box do have different application names, the Python API returns 'houdini' for either.
At least we're using the right method in case it ever get more specific.

Houdini FX
````
2017-09-07 15:03:07,396 [1724 DEBUG sgtk.env.project.tk-houdini] hou.applicationVersionString() = 16.0.705
2017-09-07 15:03:07,396 [1724 DEBUG sgtk.env.project.tk-houdini] hou.applicationName() = houdini
2017-09-07 15:03:07,396 [1724 DEBUG sgtk.env.project.tk-houdini] hou.licenseCategory() = licenseCategoryType.Commercial
2017-09-07 15:03:07,396 [1724 DEBUG sgtk.env.project.tk-houdini] hou.applicationVersion() = (16, 0, 705)
2017-09-07 15:03:07,396 [1724 DEBUG sgtk.env.project.tk-houdini] hou.applicationCompilationDate() = Aug 21 2017
2017-09-07 15:36:16,452 [4894 DEBUG sgtk.env.project.tk-houdini] hou.isApprentice()=False
```

Houdini Apprentice
```
2017-09-07 15:03:07,396 [1724 DEBUG sgtk.env.project.tk-houdini] hou.applicationVersionString() = 16.0.705
2017-09-07 15:03:07,396 [1724 DEBUG sgtk.env.project.tk-houdini] hou.applicationName() = houdini
2017-09-07 15:03:07,397 [1724 DEBUG sgtk.env.project.tk-houdini] hou.licenseCategory() = licenseCategoryType.Apprentice
2017-09-07 15:03:07,397 [1724 DEBUG sgtk.env.project.tk-houdini] hou.applicationVersion() = (16, 0, 705)
2017-09-07 15:03:07,397 [1724 DEBUG sgtk.env.project.tk-houdini] hou.applicationCompilationDate() = Aug 21 2017
2017-09-07 15:36:16,452 [4894 DEBUG sgtk.env.project.tk-houdini] hou.isApprentice()=True
```